.github/workflows/manual_system_tests.yml

```diff
@@ -135,33 +135,69 @@ jobs:
 
         exit_code=0
         SHORT_SHA=$(curl --silent -u ${{ secrets.SVC_ACCOUNT_SECRET }}:x-oauth-basic https://api.github.com/repos/0chain/$1/git/refs/heads/$2 | jq --raw-output .object.sha | head -c 8)
-        PR_ID=$(curl --silent -u ${{ secrets.SVC_ACCOUNT_SECRET }}:x-oauth-basic https://api.github.com/search/issues?q=sha:$SHORT_SHA | jq --raw-output .items[0].number)
         TAG=$(echo $2 | sed '"'"'s/\//-/g'"'"')
 
         echo "Searching for image [${TAG}-${SHORT_SHA}]..."
         docker pull 0chaindev/$3:${TAG}-${SHORT_SHA} > /dev/null 2>&1 || exit_code=$?
 
-        if [[ $exit_code == 0 ]];
-          then
-            echo "Docker image [${TAG}-${SHORT_SHA}] found"
-            echo "$(echo $3 | tr 0 Z | awk '"'"'{ print toupper($0) }'"'"')_TAG=${TAG}-${SHORT_SHA}" >> $GITHUB_ENV
-          else
-            if [ -z "$PR_ID" ];
-              then
-                echo "::error title=$3 docker image not found::$3 docker image could not be found with tag [${TAG}-${SHORT_SHA}]. Please publish image manually at https://github.com/0chain/$1/actions/workflows/build-&-publish-docker-image.yml and try again" && exit 1
-            fi
-            echo "Docker image NOT found for [${TAG}-${SHORT_SHA}]"
-            MERGE_SHORT_SHA=$(curl --silent -u ${{ secrets.SVC_ACCOUNT_SECRET }}:x-oauth-basic https://api.github.com/repos/0chain/$1/pulls/$PR_ID | jq --raw-output .merge_commit_sha | head -c 8)
-            echo "Searching for image [merge-$MERGE_SHORT_SHA]..."
-            docker pull 0chaindev/$3:merge-$MERGE_SHORT_SHA > /dev/null 2>&1 || { echo "::error title=$3 docker image not found::$3 docker image could not be found with tags [${TAG}-${SHORT_SHA}] or [merge-$MERGE_SHORT_SHA]. Please publish image manually at https://github.com/0chain/$1/actions/workflows/build-&-publish-docker-image.yml and try again" && exit 1; }
-            echo "Docker image [merge-$MERGE_SHORT_SHA] found"
-            echo "$(echo $3 | tr 0 Z | awk '"'"'{ print toupper($0) }'"'"')_TAG=merge-$MERGE_SHORT_SHA" >> $GITHUB_ENV
+        if [[ $exit_code == 0 ]];then
+          echo "Docker image [${TAG}-${SHORT_SHA}] found"
+          echo "$(echo $3 | tr 0 Z | awk '"'"'{ print toupper($0) }'"'"')_TAG=${TAG}-${SHORT_SHA}" >> $GITHUB_ENV
+
+          unset SHORT_SHA
```

Comment:
This could be moved to the start of the script which would avoid it needing to be repeated multiple times